src/tests/src/libs/antares/test_utils.cpp

```diff
@@ -0,0 +1,49 @@
+#define BOOST_TEST_MODULE test utils
+#include <boost/test/unit_test.hpp>
+
+#include <string>
+#include <antares/utils.h>
+
+
+namespace {
+
+std::string beautifyStd(const std::string& in) {
+    std::string out;
+    Antares::BeautifyName(out, in);
+    return out;
+}
```

Comment:
Those 2 methods served mainly to return the result instead of passing it as parameter: however I indeed templatized them to avoid the duplication, thanks!